cachelib/shm/SysVShmSegment.cpp

```diff
@@ -184,6 +189,18 @@ void shmCtlImpl(int shmid, int cmd, shmid_ds* buf) {
   }
 }
 
+void mbindImpl(void *addr, unsigned long len, int mode,
+               const NumaBitMask& memBindNumaNodes,
+               unsigned int flags) {
```

Comment:
nit. Why do we have this as separate function? just embed this in `memBind(...)`

Reply:
I just follow the approach used in this source file when each POSIX function is wrapped with ‘xxxImpl()’. I am nit sure why such approach was selected, perhaps, to simplify support for other OS if any.

Reply:
Thanks. I can buy that.